.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -8,6 +8,11 @@ body:
         **To report a Quarkus security vulnerability, please [send an email to `security@quarkus.io`](mailto:security@quarkus.io) with all the details.**
 
         :warning: Do **NOT** create a public issue on GitHub for security vulnerabilities. See our [security policy](https://github.com/quarkusio/quarkus/security/policy) for more details. :warning:
+  - type: markdown
+    attributes:
+      value: |
+        To maximize the chance of your issue being handled in a timely manner, please ensure you have tested that the issue occurs on the absolute latest version of Quarkus, or latest LTS version. 
+        Moreover, providing a minimal canonical sample project that makes it easy to reproduce the issue, is a huge help to the community when it comes to understanding, debugging, fixing the problem and testing the fix
```

Sorry to be that guy but: 
```suggestion
        Moreover, providing a minimal canonical sample project that makes it easy to reproduce the issue is a huge help to the community when it comes to understanding, debugging, fixing the problem and testing the fix.
```

---

Fixed :)